process/block/shardblock.go

```diff
@@ -596,7 +596,7 @@ func (sp *shardProcessor) indexBlockIfNeeded(
 		return
 	}
 
-	gasConsumedInHeader := sp.baseProcessor.gasConsumedProvider.TotalGasConsumed()
+	gasConsumedInHeader := sp.baseProcessor.gasConsumedProvider.TotalGasProvided()
```

Comment:
gasProvidedInHeader, and maybe we should change also in HeaderGasConsumption DTO (L610) from GasConsumed to GasProvided